Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.yml

```diff
@@ -48,6 +48,7 @@ configuration:
   - All Available
   - Domain Abuse
   - Vulnerability
+  - Code Repo Leakage
```

Comment:
please add this to the README of the integration

Reply:
We don't have any of the old ones in the readme, would you like me to add all?